src/middlewared/middlewared/plugins/kubernetes_linux/gpu.py

```diff
@@ -120,6 +128,29 @@
     },
 }
 
+# Contains configmaps for config files to-be-used by their respective device plugins
+GPU_CONFIGMAPS = {
+    'NVIDIA': {
+        'apiVersion': 'v1',
+        'kind': 'ConfigMap',
+        'metadata': {
+            'name': 'nvidia-device-plugin-config',
+            'namespace': 'kube-system'
+        },
+        'data': {
+            'nvdefault.yaml': """version: v1
```

Comment:
@stavros-k and I where not sure if we needed to add `|` here, which is required for multiline in kube yaml.

Reply:
```
Allocatable:
  cpu:                16
  ephemeral-storage:  612650699501
  hugepages-1Gi:      0
  hugepages-2Mi:      0
  memory:             65804624Ki
  nvidia.com/gpu:     5
  pods:               250
  ```
  
GPU changes ^^^

Reply:
Ahh you mean you already tested? :)

Reply:
Ofc ✅

Reply:
I stand corrected, good to know python multiline syntax translates correctly to kube yaml...
Will make note for next time! :)